serverless/serverless-components
# Add input to specify lambda package root

## What has been implemented?

Fixes: https://serverlessteam.atlassian.net/browse/SC-193

This adds a `root` input to the `aws-lambda` component which allows the user to specify which directory will be zipped up and uploaded to AWS Lambda.

Before this, the `aws-lambda` component zipped up all files in your top-level component.  For example, in the `retail-app` component all files were being uploaded to AWS Lambda (e.g., the front-end assets).

This also updates the retail-app component's serverless.yml to NOT zip up everything.

## Steps to verify

* Deploy the retail app
* Go to the AWS Lambda console and see if your functions contain only the items in the `code` folder.
